.mend/plans/ISSUE-246.md

```
@@ -0,0 +1,90 @@
+# ISSUE-246: Missing feature file: package_check.feature
+
+## Problem Statement
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)